packages/flutter/test/foundation/error_reporting_test.dart

```diff
@@ -70,22 +70,22 @@ Future<void> main() async {
       },
     ));
     expect(console.join('\n'), matches(
-      '^══╡ EXCEPTION CAUGHT BY ERROR HANDLING TEST ╞═══════════════════════════════════════════════════════\n'
-      'The following assertion was thrown testing the error handling logic:\n'
-      'Message goes here\\.\n'
-      '\'[^\']+flutter/test/foundation/error_reporting_test\\.dart\':\n'
-      'Failed assertion: line [0-9]+ pos [0-9]+: \'false\'\n'
-      '\n'
-      'When the exception was thrown, this was the stack:\n'
-      '#0      getSampleStack\\.<anonymous closure> \\([^)]+flutter/test/foundation/error_reporting_test\\.dart:[0-9]+:[0-9]+\\)\n'
-      '#2      getSampleStack \\([^)]+flutter/test/foundation/error_reporting_test\\.dart:[0-9]+:[0-9]+\\)\n'
-      '#3      main \\([^)]+flutter/test/foundation/error_reporting_test\\.dart:[0-9]+:[0-9]+\\)\n'
-      '(.+\n)+' // TODO(ianh): when fixing #4021, also filter out frames from the test infrastructure below the first call to our main()
-      '\\(elided [0-9]+ frames from class _RawReceivePortImpl and dart:async\\)\n'
-      '\n'
-      'line 1 of extra information\n'
-      'line 2 of extra information\n'
-      '════════════════════════════════════════════════════════════════════════════════════════════════════\$',
+      r'^══╡ EXCEPTION CAUGHT BY ERROR HANDLING TEST ╞═══════════════════════════════════════════════════════\n'
```

Comment:
These `\n`s  will now be `\\n`.  I don't think we can do this one here (and below).

I also find it strange that Dart would let me concat a raw string with a non-raw string this way...

Reply:
> These `\n`s will now be `\\n`. I don't think we can do this one here (and below).

I think the result will be the same.

```dart
RegExp('\n').hasMatch('\n'); // true
RegExp('\\n').hasMatch('\n'); // true
```

In the first case `\n` is _expanded_ by the Dart language. In the second case it is the regex engine that handle the `\n`. Finally it seems to have the same result.

Reply:
Ahhh I missed that these are regexp.  Ok.